config.yaml

```diff
@@ -150,6 +150,9 @@ params:
       - name: notaryproject.dev on GitHub
         url: https://github.com/notaryproject/notaryproject.dev
         icon: fab fa-github
+      - name: Notary v2 Conversations on Slack
+        url: https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/thread/CEX1W7WMD-1582660575.076600
```

Comment:
Linking to the Notary V2 channel may be better than linking to a specific thread here:
https://cloud-native.slack.com/archives/CQUH8U287

Reply:
Okay
I have fixed the DCO.
I will fix the link also once it is approved from @SteveLasker .

Reply:
yup, i was going to say the same thing. I just needed to get to a full computer to verify the link

Reply:
Actually, reading @nate-double-u 's comments, I would agree. Centralizing these in the community page is better as it becomes fewer places to update as things evolve. And, they always do.
Maybe, a "community" link, or "get involved" link that points to the community page?

Reply:
@SteveLasker @nate-double-u 
Can you please check if this works for you all or not??
Recording link: https://www.loom.com/share/b814f3b384474028aac6e8d7d56a0bb4